custom.ini

```diff
@@ -0,0 +1 @@
+post_max_size=128M
```

Comment:
What does this do?

Reply:
It increases the size limit of POST data (file upload) that the server will accept. The default value is usually 8 megabytes. This value is a bit excessive. I'll reduce it to something like `16M` or `32M`